src/components/indicators/indicators.js

```diff
@@ -15,37 +16,33 @@ define(['datetime', 'itemHelper', 'emby-progressbar', 'css!./indicators.css', 'm
         return false;
     }
 
-    function getProgressHtml(pct, options) {
-        var containerClass = 'itemProgressBar';
-        if (options) {
-            if (options.containerClass) {
-                containerClass += ' ' + options.containerClass;
-            }
+    export function getProgressHtml(pct, options) {
+        let containerClass = 'itemProgressBar ';
+        if(options && options.containerClass) {
+            containerClass += options.containerClass
         }
 
         return '<div class="' + containerClass + '"><div class="itemProgressBarForeground" style="width:' + pct + '%;"></div></div>';
     }
 
     function getAutoTimeProgressHtml(pct, options, isRecording, start, end) {
-        var containerClass = 'itemProgressBar';
-        if (options) {
-            if (options.containerClass) {
-                containerClass += ' ' + options.containerClass;
-            }
+        let containerClass = 'itemProgressBar ';
+        if(options && options.containerClass) {
+            containerClass += options.containerClass
         }
 
-        var foregroundClass = 'itemProgressBarForeground';
+        let foregroundClass = 'itemProgressBarForeground ';
```

Comment:
Good catch. But I would leave `foregroundClass` without spaces, i.e. would move it under condition.

Reply:
3 suggestions are for just spaces :P

Reply:
I am very picky. :p

Reply:
done the needful :)